src/main/java/pl/simpleascoding/tutoringplatform/advertisement/AdvertisementController.java

```diff
@@ -7,10 +7,9 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
-import pl.simpleascoding.tutoringplatform.dto.AdvertisementDTO;
-import pl.simpleascoding.tutoringplatform.dto.CreateAdvertisementDTO;
-import pl.simpleascoding.tutoringplatform.dto.RscpDTO;
-import pl.simpleascoding.tutoringplatform.service.advertisement.AdvertisementService;
+import pl.simpleascoding.tutoringplatform.advertisement.dto.Advertisement;
+import pl.simpleascoding.tutoringplatform.advertisement.dto.CreateAdvertisement;
+import pl.simpleascoding.tutoringplatform.util.rscp.RscpDTO;
 import pl.simpleascoding.tutoringplatform.util.ControllerUtils;
 
```

Comment:
we should discuss the controller accessibility

Reply:
controller accessibility will be reduced to package-private.